apps/ensrainbow/package.json

```diff
@@ -23,7 +23,10 @@
     "docker:push": "docker push ensnode/ensrainbow",
     "test": "DATA_DIR=test-data vitest run",
     "test:watch": "DATA_DIR=test-data vitest",
-    "test:coverage": "DATA_DIR=test-data vitest run --coverage"
+    "test:coverage": "DATA_DIR=test-data vitest run --coverage",
+    "lint": "biome check --write .",
+    "lint:ci": "biome ci",
+    "typecheck": "tsc --noEmit"
```

Comment:
Our CI setup looks for `lint:ci` & `typecheck` commands across monorepo moduels:

https://github.com/namehash/ensnode/blob/a38060c58e2547d780a8b75e1a75ee55511727b5/.github/workflows/ci.yml#L33-L37